Dockerfile.format

```
@@ -3,7 +3,7 @@ FROM openjdk:15-alpine
 RUN mkdir -p /app
 WORKDIR /app
 
-ADD https://github.com/google/google-java-format/releases/download/google-java-format-1.9/google-java-format-1.9-all-deps.jar /app/google-java-format.jar
+ADD https://github.com/google/google-java-format/releases/download/v1.15.0/google-java-format-1.15.0-all-deps.jar /app/google-java-format.jar
```

```suggestion
ADD https://github.com/google/google-java-format/releases/download/v1.15.0/google-java-format-1.15.0-all-deps.jar google-java-format.jar
```
Since we're changing `JAR_FILE` path, right?